scripts/test.js

```diff
@@ -3,6 +3,7 @@ const cp = require("child_process");
 const path = require("path");
 const fs = require("fs");
 const os = require("os");
+const glob = require("glob");
```

Comment:
Where this dependency comes from?

Reply:
Added in https://github.com/rescript-lang/rescript/pull/7355/commits/91ec3f8778567996b8ba9dce281c3d881cbd6c09

Reply:
We can use a better alternative for globbing

https://github.com/es-tooling/module-replacements/blob/main/docs/modules/glob.md

However, I'm not sure this is a good approach to go

Reply:
I think `glob` seems to be enough.

Reply:
Removed